tests/defer/router/subgraphs/computers/computers.js

```diff
@@ -0,0 +1,35 @@
+const {ApolloServer, gql} = require('apollo-server');
+const {readFileSync} = require('fs');
+
+const port = process.env.APOLLO_PORT || 4000;
+
+const computers = [
+    {id: 'Computer1', cpu: "386", year: 1993, screen: {resolution: "640x480", isColor: false}},
+    {id: 'Computer2', cpu: "486", year: 1996, screen: {resolution: "800x600", isColor: true}},
+]
+
+const typeDefs = gql(readFileSync('./computers.graphqls', {encoding: 'utf-8'}));
+const resolvers = {
+    Query: {
+        computers: (_, args, context) => {
+            return computers;
+        },
+        computer: (_, args, context) => {
+            return computers.find(p => p.id === args.id);
+        }
+    },
+    Computer: {
+        errorField: (_, args, context) => {
+            throw new Error("Error field");
+        },
+        nonNullErrorField: (_, args, context) => {
+            throw new Error("Error field");
```

Comment:
Nit: in the reference implementation tests, the `nonNullErrorField` [attempts to return `null`](https://github.com/graphql/graphql-js/blob/a24a9f35b876bdd0d5050eca34d3020bd0db9a29/src/execution/__tests__/defer-test.ts#L56-L58)